templates/dockerhub/Dockerfile

```diff
@@ -28,9 +28,11 @@ LABEL version="<%= version %>" \
 VOLUME ["/actions"]
 WORKDIR /actions
 
-COPY --from=build /tmp/fakeroot/ /
+RUN mkdir -p /etc/ssl/certs
+COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
-## The loader search ld-linux-x86-64.so.2 in /lib64 but the folder does not exist
-RUN ln -s lib lib64
+COPY --from=build /tmp/fakeroot/ /
+COPY --from=ghcr.io/tarampampam/curl:8.6.0 /bin/curl /usr/bin/curl
```

Comment:
question: why not use `apt-get` instead?

Reply:
And copy it from the `build` image ?
By the way, I add curl to this image and I can get script or call we hoot inside my CI.